tests/application-connector-tests/Dockerfile

```diff
@@ -1,13 +1,14 @@
-FROM golang:1.10-alpine as builder
+FROM golang:1.13-alpine as builder
 
 ENV SRC_DIR=/go/src/github.com/kyma-project/kyma/tests/application-connector-tests
+ENV CGO_ENABLED=0
```

Comment:
I think we can do it the same as in other components, setting this var only for building binary like this: `RUN CGO_ENABLED=0 go test...`

Reply:
Done!